### DISCUSSION TITLE
Automatic export error

### DISCUSSION THREAD

**Discussion by BeatTraxx** [Xzandro/sw-exporter#397] [Q&A] [answered]
Hi I am having error messaged when I activate automatic export to swarfarm all the time. 
I triple checked the API key and even generated new ones just to be sure but it still doesn't work. 

Any ideas how to fix that? 
I am using MuMu emulator if this is important. 
![asdasdasd](https://user-images.githubusercontent.com/19388452/123341665-a5c13480-d54e-11eb-9962-e5c3a401f903.png)

**Comment by Xzandro** [comment, marked as answer]
Since this is generated by the SwarfarmLogger, I would recommend to ask the SWARFARM devs directly about this issue.

**Reply by BeatTraxx** [reply]
Oh, I see, thank you!
